### PR TITLE
Get api api endpoint from env

### PR DESCRIPTION
**Pull request by mcchang**

### CODE REVIEW

File: frontend/src/api/ClimateChangemakersAPI.ts [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import { ActionInfo } from "../models/ActionInfo";
 import { IssuesResponse } from "../models/IssuesResponse";
 
+// const apiEndpoint = "https://changemakers-act-backend.onrender.com/"
+const apiEndpoint = process.env.REACT_APP_API_ENDPOINT || '';
```
Comment (kevincianfarini) [inline frontend/src/api/ClimateChangemakersAPI.ts:5]: How does this impact the anticipated bundling of the frontend into static assets which will be served from somewhere? We're not going to have an environment at that point, and the `apiEndpoint` is going to implicitly be where this content was served from, right?

File: frontend/src/api/ClimateChangemakersAPI.ts [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+console.log(process.env.NODE_ENV);
```
Comment (jacknugent) [inline frontend/src/api/ClimateChangemakersAPI.ts:6]: Dev code?

File: frontend/src/api/ClimateChangemakersAPI.ts (hunk 1 above)
Comment (mcchang) [reply frontend/src/api/ClimateChangemakersAPI.ts:5]: Yeah, ignore this PR.  It don't think it's needed.  I was just experimenting with various ways to deploy the frontend and the backend.

File: frontend/src/api/ClimateChangemakersAPI.ts (hunk 2 above)
Comment (mcchang) [reply frontend/src/api/ClimateChangemakersAPI.ts:6]: yeah, this PR was a WIP.  Disregard!

**Pull request closed without merging**